webflow-hacks/interactions-hacks/10.ts

```diff
@@ -0,0 +1,48 @@
+// not needed if using cdn
+// import * as Cookies from 'js-cookie';
```

Comment:
@aexshafii You just have to add `"types": ["js-cookie"]` in the `tsconfig.json` file.
It'll automatically detect the types.

Reply:
👌